### PR TITLE
lnrpc: fix comment typo

### PR DESCRIPTION
**Pull request by ottosuess**
i guess "creted" should be "created"

### CODE REVIEW

File: lnrpc/rpc.proto [hunk 1]
```diff
@@ -1013,7 +1013,7 @@ message Channel {
     /// Whether this channel is advertised to the network or not.
     bool private = 17 [json_name = "private"];
 
-    /// True if we were the ones that creted the channel.
+    /// True if we were the ones that created the channel.
```
[the remark this answers is not in the document]
Comment (cfromknecht) [reply lnrpc/rpc.proto]: good call!

**Comment by halseth** [comment]
Looks good! Can you squash the commits into one? :)

**Comment by ottosuess** [comment]
squashed 🙌

**Comment by halseth** [comment]
`make rpc` had to be run with the correct proto version for the file to be generated correctly, so I did that for you: https://github.com/lightningnetwork/lnd/pull/2634

**Pull request closed without merging**